### DISCUSSION TITLE
[need help]Music not displayed correct.

### DISCUSSION THREAD

**Discussion by LFS96** [jellyfin/jellyfin#12216] [General]
Hello,

I am trying to switch from Plex to Jellyfin, but my music folder is not displayed as I would like it. 
I’ll give you an example below:

Plex:
![grafik](https://github.com/jellyfin/jellyfin/assets/26200357/03627f7e-8d34-4929-80ed-cd3aff6a1514)

Jellyfin:
![grafik](https://github.com/jellyfin/jellyfin/assets/26200357/c3c44b7f-1536-4aa0-91f4-8193ac7f25f3)
![grafik](https://github.com/jellyfin/jellyfin/assets/26200357/18b29623-0f9a-4ba1-897a-cf3f803c88a1)

As you can see, Jellyfin displays only one album with 41 hours (all tracks found). When I start playing a song, the correct album is recognized.

Have I missed some step in the instructions, or is this a common bug? I am using the most up-to-date Jellyfin version from Docker Hub. I’ve tried importing my library on two PCs with the same result. My library is about 100GB.

**Comment by TheMelmacian** [comment]
What is your folder structure?

**Reply by LFS96** [reply]
My folder structure is: First_Letter_of_Album_Artist/Album_Artist/Album/[{CD}CD_Number/]Title.mp3

For example:
A/ASP/Horrors/04 Klick.mp3

The CD_Number is optional; I use it only if there are multiple CDs in one album.

Besides the folder structure, all MP3 songs have tags to identify Album_Artist, Artist, Title, Track, CD_Number, Max_CD, Genre, and Release_Year.

**Reply by TheMelmacian** [reply]
Jellyfin works best if the folder structure is: _Artist/Album/Track.ext_
Also it is recommended to not use sub folders for discs but to use tags to define which track is part of which cd.
see: https://jellyfin.org/docs/general/server/media/music/

In your library settings instead of adding the root path of your music library, try to add all sub folders:
- path/to/your/musiclibrary/A
- path/to/your/musiclibrary/B
- path/to/your/musiclibrary/C
- ...
